core/drivers/stm32_rng.c

```diff
@@ -283,6 +286,10 @@ static TEE_Result stm32_rng_probe(const void *fdt, int offs,
 	    rstctrl_deassert_to(stm32_rng->rstctrl, RNG_RESET_TIMEOUT_US))
 		goto err;
 
+	res = init_rng();
+	if (res)
+		goto err;
```

Comment:
need to disable the clock, for consistency.

Reply:
Yes